### DISCUSSION TITLE
configure

### DISCUSSION THREAD

**Discussion by Monica-beta** [silveredgold/beta-protection#35] [General]
Cannot configure overdrive. One has to spend a whole night repeating the configuration until one decides to accept it.
Now I wanted to activate the writing but nothing does not accept the modification of the times, it is not activated. are things to fix

**Comment by silveredgold** [comment]
Sorry, I really don't understand you. Can you try covering more clearly:

- What are you trying to achieve?
- What have you done so far?
- What's not working, and what errors/messages are you getting when you try?

**Reply by b2sbeta** [reply]
Sounds like they're trying to configure an override but aren't too clear about how it works. 

@Monica-beta, did you read the documentation: https://silveredgold.github.io/beta-protection/guide/overrides.html#how-does-it-work
